README.md

```diff
@@ -91,6 +91,18 @@ Open the dashboard in a browser:
 
 `minikube dashboard`
 
+## Workaround for connection issues when using a VPN
+
+Certain VPNs (on the host machine) have known issues with port conflicts when using the Hyperkit driver. The Hyperkit
+driver is used when running on an x86-64 machine.
+
+If disabling the VPN is not possible, one workaround is to use the `docker` driver along with the `docker-mac-net-connect` brew service.
```

Comment:
This workaround seems macOS specific. Can we mention this in the title or intro ?

Reply:
Good point - I've update the title.